exercises/rna-transcription/.meta/template.j2

```diff
@@ -0,0 +1,26 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual(
+            {{ case["property"] | to_snake }}('{{ input["dna"] }}'),
+            '{{ case["expected"] }}'
+        )
+{%- endmacro %}
+{{ macros.header()}}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    {{ test_case(case) }}
+    {% endfor %}
+    {% if additional_cases | length -%}
+
+    # Additional tests for this track
+
+    {% for case in additional_cases -%}
+    {{ test_case(case) }}
+    {% endfor %}
+    {%- endif %}
```

Comment:
This section for additional tests may be removed; there are no such tests in this exercise.

Reply:
I just removed this section. As the section was surrounded by an if additional_cases, I thought that would be OK.

Reply:
I didn't technically hurt, it was just code that never runs, and doesn't need to be there.